op-chain-ops/crossdomain/legacy_withdrawal.go

```diff
@@ -57,7 +57,9 @@ func (w *LegacyWithdrawal) Encode() ([]byte, error) {
 	return out, nil
 }
 
-// Decode will decode a serialized LegacyWithdrawal
+// Decode will decode a serialized LegacyWithdrawal. There is a known inconsistency
```

Comment:
Can we add a panic in the `Encode` method as well to prevent improper usage elsewhere?

Reply:
I've added this now